Travel-list/src/App.jsx

```diff
@@ -1,13 +1,50 @@
+import { useState } from "react";
 import "./App.css";
 import FormSection from "./components/FormSection";
 import Header from "./components/Header";
+import PackingList from "./components/PackingList";
+import Stats from "./components/Stats";
 
 function App() {
+  const [itemObject, setItemObject] = useState([
+    {
+      Quantity: 1,
+      Item: "Charger",
+    },
+  ]);
+  const [packedItems, setPackedItems] = useState(
+    Array(itemObject.length).fill(false)
+  );
+
+  const addItemToItemObject = (newItem) => {
+    setItemObject([...itemObject, newItem]);
+    setPackedItems([...packedItems, false]);
+  };
```

Comment:
we don't need packedItems state.. Also `addItemToItemObject`.. Can it be named something else because if we don't add item to itemObject, where else we will.

So, I think a better name should be just `addItem`.